packages/manager/apps/public-cloud/package.json

```diff
@@ -58,7 +58,7 @@
     "oclazyload": "^1.1.0",
     "ovh-angular-pagination-front": "ovh-ux/ovh-angular-pagination-front#^5.1.0",
     "ovh-angular-q-allsettled": "ovh-ux/ovh-angular-q-allSettled#^0.3.1",
-    "ovh-api-services": "^7.0.0",
+    "ovh-api-services": "^7.3.1",
```

Comment:
You'll need to bump for all `apps` 
`yarn upgrade-interactive --latest ovh-api-services`

Reply:
This is done. Thanks for letting me know :)